SubmittedAssignments/sanjeevanee_balure_66/com..sty..css

```
@@ -0,0 +1,149 @@
+			index: x-index;
+			}
+		.certificate .recttop4 {
+		position: absolute;
+		    top: 0pt;
+			left: 0pt;
+            width: 140pt;
+            height: 30pt;
+            background-color: #002147;
+			border: 2pt solid goldenrod;
+            border-radius: 0pt 0pt 20pt 0pt;
+			index: z-index
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)